### DISCUSSION TITLE
Help how to refresh configuration v1 state?

### DISCUSSION THREAD

**Discussion by Marlo461** [zwave-js/zwave-js-ui#2712] [Q&A]
Hello everyone, I need some advice. I use Fibaro FGT-001 heads for heating with radiators. They work well, but I want to use the function (heating medium demand reports) that this head has and read its status (provide heat in order to maintain set temperature). With this state, I want to control the gas boiler. I encountered a problem that is probably caused by the fact that this head is a Z-Wave FLiRS. The problem is that the header changes the status of the request, but the status is sometimes sent and sometimes not. I tested it for a few days and found out that even when the thermostat is OFF, this request is sometimes active, but when the Z-Wave JS UI on this device was manually restored (configuration v1), this status is loaded correctly. My question is whether I know how to create an automation that would perform status recovery (configuration v1). Please advise how I can request a renewal using automation or a script.
I am also attaching a photo before and after the refresh.
[Before refresh configuration v1](https://files.fm/f/mjk76csra)
[After refresh configuration v1](https://files.fm/f/aphm4uu7p)

**Comment by robertsLando** [comment]
The only way is to use refresh command class api, there is a button in ui too near the expand button on every Command class title. Another alternative is to setup value poll in settings. 

Are you using it with mqtt or home assistant?

**Reply by Marlo461** [reply]
Home assistant

**Reply by robertsLando** [reply]
@Marlo461 I dunno how you could refresh a value using hass but you could setup [scheduled poll](https://zwave-js.github.io/zwave-js-ui/#/usage/setup?id=device-values-configuration) on z-ui to keep it updated and then I let @kpine or @blhoward2 help you with HA setup

**Reply by Marlo461** [reply]
OK thanks. Scheduled polling works fine, (I've tested now) but probably drained my battery quickly because it will call even when the thermostat is off. If only it could be called in 5 minute intervals when the thermostat is on

**Reply by robertsLando** [reply]
@Marlo461 The alternative could be to setup a node-red flow or else that does the refresh periodically only when the thermostat is on

**Reply by blhoward2** [reply]
If polling works, you can refresh the value with a service call in HA. Just setup an automation to do that when the thermostat is on with whatever frequency you like.

**Reply by Marlo461** [reply]
I just don't know how to call poll for configuration v1.  node red, if you could help me how to do it, that would be great

**Reply by blhoward2** [reply]
Either way you'd be using the same service call. Just do it in an automation in HA. There is a refresh value service under zwavejs. Have it run every 5 minutes. Condition that the thermostat is on.

**Reply by Marlo461** [reply]
![Screenshot_20221010-150811_Home Assistant.jpg](https://user-images.githubusercontent.com/96550831/194873694-9a0f29f9-c85f-485a-ad35-3a2f1e7b1831.jpg)

This refresh?

**Reply by blhoward2** [reply]
Yes

**Reply by Marlo461** [reply]
this does not work for configuration v1 where this information is.  i already tried it.  restores only temperature values, battery...
![Screenshot_20221010-175942_Home Assistant.jpg](https://user-images.githubusercontent.com/96550831/194907886-edf8d835-0255-43d0-8870-df67cf6cc063.jpg)

**Reply by blhoward2** [reply]
It calls the exact same function call in the driver as zwave-js-ui calls during its polling. And the same as hitting refresh on the device. If you mean the values are in a configuration parameter, that's not a traditional poll. 

@robertsLando can your polling refresh a parameter?

**Reply by Marlo461** [reply]
![Screenshot_20221010-182117_Home Assistant_1.jpg](https://user-images.githubusercontent.com/96550831/194912071-65e31af5-a226-4730-bf64-edf294195b10.jpg)
![Screenshot_20221010-182100_Home Assistant_1.jpg](https://user-images.githubusercontent.com/96550831/194912084-a88fcd5b-52f6-47b4-8637-1b3840fa3262.jpg)

**Reply by kpine** [reply]
You can refresh the configuration parameter value:

```yaml
service: zwave_js.invoke_cc_api
data:
  command_class: "112"
  method_name: get
  parameters:
    - 3
target:
  device_id: 6e0210ecabaa59d9f08891c8db40e127
```

**Reply by blhoward2** [reply]
I only read from where I was tagged and didn't realize we were talking about a parameter. @kpine has the way.

**Reply by Marlo461** [reply]
thank you this works 👍
